### PR TITLE
tests: update for consistent code style

### PR DESCRIPTION
**Pull request by miabbott**
This change just reflows some of the uses of `role:` to match the
style we have used elsewhere.  I think this covers all the 'bad' style
uses.

### CODE REVIEW

**Comment by miabbott** [comment]
[(3) Style cleanup of [a-h-t] playbooks](https://trello.com/c/NRjhvKW6/686-3-style-cleanup-of-a-h-t-playbooks)

**Comment by miabbott** [comment]
[(3) Style cleanup of [a-h-t] playbooks](https://trello.com/c/NRjhvKW6/686-3-style-cleanup-of-a-h-t-playbooks)

File: tests/improved-sanity-test/main.yml [hunk 1]
```diff
@@ -240,7 +272,11 @@
 
   roles:
     # This playbook requires Ansible 2.1 and an Atomic Host
-    - { role: ansible_version_check, avc_major: "2", avc_minor: "0", tags: ['ansible_version_check'] }
+    - role: ansible_version_check
+      avc_major: "2"
+      avc_minor: "0"
```
Comment (mike-nguyen) [inline tests/improved-sanity-test/main.yml]: avc_minor: "1" ?

**Comment by miabbott** [comment]
Our Red Hat CI checks are going to fail because of the `default_t` problem.  I don't think that is related to this change.

**Comment by mike-nguyen** [comment]
I ran it with --skip-tags selinux_verify to test the rest of it and it ran fine.  Merging it!

**Pull request merged**